torch_xla/csrc/BUILD

```diff
@@ -150,6 +150,7 @@ ptxla_cc_library(
         "@org_tensorflow//tensorflow/core/lib/core:errors",
         "@org_tensorflow//tensorflow/core/lib/gtl:inlined_vector",
         "@org_tensorflow//tensorflow/core/profiler/lib:traceme",
+        "@org_tensorflow//tensorflow/core/framework:kernel_shape_util",
```

Comment:
Why do you have `"@org_tensorflow//tensorflow/compiler/tf2xla/kernels:conv_op_helpers"` on line 131 above?

Reply:
Thanks for pointing that, Milad, updated

Reply:
@ManfeiBai please confirm there is "other" use of `"@org_tensorflow//tensorflow/compiler/tf2xla/kernels:conv_op_helpers"` in our BUILD files or in our `.h`/`.cpp` files.

Reply:
Thanks, Milad, has checked and updated in the ticket